src/@types/event.ts

```diff
@@ -33,6 +33,7 @@ export enum EventType {
     RoomGuestAccess = "m.room.guest_access",
     RoomServerAcl = "m.room.server_acl",
     RoomTombstone = "m.room.tombstone",
+    RoomFunctionalMembers = "io.element.functional_members",
```

Comment:
Ideally only `m.` prefixed items live in the js-sdk for a clean separation but there is a single other `io.element` instance already here

Reply:
`UnstablePrefix` should help a lot here.

Reply:
@turt2live it isn't unstable though, it seems to be Element-specific functionality which doesn't appear to be intended for MSC'ing

Reply:
Correct, for now we don't have plans to standardise this.
If it proves to be useful to a wider audience, I'm sure an MSC will follow at some point.

Reply:
From a surface glance it'll likely be handed in standard by https://github.com/matrix-org/matrix-doc/blob/travis/msc/immutable-dms/proposals/2199-canonical-dms.md `unimportant` users

Reply:
Anything off-spec is considered unstable by the spec, so it would still apply here. You don't need to specify a stable value to `UnstablePrefix`

Reply:
Or it might be `UnstableNamespace`, but either way a class exists for this exact reason

Reply:
This is my first PR to the repo. Can you point me to the place of unstable event types?
I see one UPPER_CASE unstable constant at the bottom of `event.ts`. Is that where this type is supposed to go?

Reply:
It's not a well-defined system just yet, but something like that would be best: goal is to not have it in the enum until it is a stable identifier, at which point there's two references to it while people transition to the enum.

Reply:
I renamed it to `UNSTABLE_ELEMENT_FUNCTIONAL_USERS`.